docs/user-guide/cli-configuringcli-ev.md

```diff
@@ -32,6 +32,5 @@ By default, the CLI daemon mode binary creates or reuses a file in the user's ho
 
 | Platform | Environment Variable  | Description | Values | Default |
 | ---------------------- | ---------------------- | ---------------------- | ---------------------- | ---------------------- |
-| Windows | `ZOWE_DAEMON_LOCK` | Specifies an alternative path to the lock file that restricts access to the named pipe that the daemon uses for communication. | Any valid path on your computer | `%HOMEPATH%\.zowe-daemon.lock` |
-| Linux, macOS | `ZOWE_DAEMON` | Specifies an alternative path to the socket that the daemon uses for communication. | Any valid path on your computer | `$HOME/.zowe-daemon.sock` |
-| Windows | `ZOWE_DAEMON` | Specifies an alternative name for the named pipe that the daemon uses for communication. The environment variable cannot be defined as a file path that resides on a disk. | Any valid Windows pipe name without the `\\.\` prefix | `%USERNAME%\ZoweDaemon` |
\ No newline at end of file
+| All | `ZOWE_DAEMON_DIR` | Lets you override the complete path to the directory that will hold daemon files related to this user. The directory can contain the following files: <ul><li>`daemon.lock`</li><li>`daemon.sock`</li><li>`daemon_pid.json`</li></ul> | Any valid path on your computer | `%HOMEPATH%/.zowe/daemon` |
```

Comment:
I do not know what you usually do about specifications for Windows vs Linux.  I can follow where the demon directory is from the existing specification of:

    $HOMEPATH%/.zowe/daemon

However, HOMEPATH only exists on Windows, and backslashes (\\) are used on Windows instead of slashes (/). 

The following specifications contain the correct details:

    Windows: %HOMEPATH%\.zowe\daemon

    Linux: $HOME/.zowe/daemon

You can decide if you want to show both, or show just one and say "for example on Linux', or use some generalized description like 'Your home directory'. If you generalize the home directory the slashes vs backslashes are probably less important. People tend to know which type of slash to use on their computer.